src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -0,0 +1,105 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { Avatar, DataList, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+
+export interface IAssetDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The source of the logo for the token.
```

Comment:
Let's always refer to asset instead of token on this prop comment and on the ones below
```suggestion
     * The logo source of the asset
```